CRM/Civicase/Hook/NavigationMenu/AlterForCaseMenu.php

```diff
@@ -119,7 +122,7 @@ private function getCaseCategoryName($url) {
     $urlParams = parse_url(htmlspecialchars_decode($url), PHP_URL_QUERY);
     parse_str($urlParams, $urlParams);
 
-    return !empty($urlParams['case_type_category']) ? $urlParams['case_type_category'] : 'Cases';
+    return !empty($urlParams['ctc']) ? $urlParams['ctc'] : 'Cases';
```

Comment:
Can you use the constant for the name here: CRM_Civicase_Helper_CaseCategory::CASE_TYPE_CATEGORY_NAME